pyftpdlib/handlers.py

```diff
@@ -3449,6 +3453,15 @@ def __init__(self, conn, server, ioloop=None):
 
         def __repr__(self):
             return FTPHandler.__repr__(self)
+            
```

Comment:
`flake8` (which is run by the Travis builds) is complaining about this line being blank, but still containing space characters